module/proto/peggy/v1/claims.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+package peggy.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/althea-net/peggy/module/x/peggy/types";
+
+enum ClaimType {
+  UNKNOWN             = 0;
+  DEPOSIT             = 1;
+  WITHDRAW            = 2;
+  ERC20_DEPLOYED      = 3;
+  LOGIC_CALL_EXECUTED = 4;
+}
+
+// EthereumBridgeDepositClaim
+// When more than 66% of the active validator set has
+// claimed to have seen the deposit enter the ethereum blockchain coins are
+// issued to the Cosmos address in question
+// -------------
+message DepositClaim {
```

Comment:
ideally, all the fields should have a comment for reference

Reply:
we can add this later